server/fishtest/static/js/application.js

```diff
@@ -1,102 +1,116 @@
-    : v1 !== '' && v2 !== '' && !isNaN(p1 = padDotVersion(v1)) && !isNaN(p2 = padDotVersion(v2))
-    ? p1 - p2
-    : v1 !== '' && v2 !== '' && !isNaN(padDotVersion(v1.replace('clang++ ', '').replace('g++ ', ''))) && !isNaN(padDotVersion(v2.replace('clang++ ', '').replace('g++ ', '')))
-    ? padDotVersionStr(v1).toString().localeCompare(padDotVersionStr(v2))
-    : v1.toString().localeCompare(v2)
-)(getCellValue(asc ? a : b, idx), getCellValue(asc ? b : a, idx));
+const comparer = (idx, asc) => (a, b) =>
+  ((v1, v2) =>
+    v1 !== "" && v2 !== "" && !isNaN(v1) && !isNaN(v2)
+      ? v1 - v2
+      : v1 !== "" && v2 !== "" && !isNaN("0x" + v1) && !isNaN("0x" + v2)
+      ? parseInt(v1, 16) - parseInt(v2, 16)
+      : v1 !== "" &&
+        v2 !== "" &&
+        !isNaN((p1 = padDotVersion(v1))) &&
+        !isNaN((p2 = padDotVersion(v2)))
+      ? p1 - p2
+      : v1 !== "" &&
+        v2 !== "" &&
+        !isNaN(padDotVersion(v1.replace("clang++ ", "").replace("g++ ", ""))) &&
+        !isNaN(padDotVersion(v2.replace("clang++ ", "").replace("g++ ", "")))
+      ? padDotVersionStr(v1).toString().localeCompare(padDotVersionStr(v2))
+      : v1.toString().localeCompare(v2))(
+    getCellValue(asc ? a : b, idx),
+    getCellValue(asc ? b : a, idx)
+  );
 
-$(() => {
-    // clicking on table headers sorts the rows
-    $(document).on('click', 'th', (event) => {
-        const th = $(event.currentTarget)[0];
-        const table = th.closest('table');
-        const body = table.querySelector('tbody');
-        Array.from(body.querySelectorAll('tr'))
-            .sort(comparer(Array.from(th.parentNode.children).indexOf(th), this.asc = !this.asc))
-            .forEach(tr => body.appendChild(tr));
-    });
-
-    $("#non_default-button").click(function() {
-        const active = $(this).text().trim().substring(0, 4) === 'Hide';
-        $(this).text(active ? 'Show non default nets' : 'Hide non default nets');
-        $.cookie('non_default_state', active ? 'Hide' : 'Show', {expires: 3650});
-        window.location.reload();
-    });
-
-    $("#machines-button").click(function() {
-        const active = $(this).text().trim() === 'Hide';
-        $(this).text(active ? 'Show' : 'Hide');
-        $("#machines").slideToggle(150);
-        $.cookie('machines_state', $(this).text().trim(), {expires: 3650});
-    });
+document.addEventListener("DOMContentLoaded", () => {
+  document.addEventListener("click", (e) => {
+    const { target } = e;
+    if (target.matches("th")) {
+      const th = target;
+      const table = th.closest("table");
+      const body = table.querySelector("tbody");
+      Array.from(body.querySelectorAll("tr"))
+        .sort(
+          comparer(
+            Array.from(th.parentNode.children).indexOf(th),
+            (this.asc = !this.asc)
+          )
+        )
+        .forEach((tr) => body.appendChild(tr));
+    }
+  });
+  const nonDefaultButton = document.querySelector("#non_default-button");
```

Comment:
I'm wondering if it's possible to move the `#non_default-button` code in `nns.mak` like `"#machines-button"` (code moved in `tests.mak`) and `"#tasks-button"` (code moved in `tests_view.mak`)